components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -3760,6 +3760,13 @@ File path for RRTMGP longwave gaseous absorption coefficients file.
 File path for RRTMGP shortwave gaseous absorption coefficients file.
 </entry>
 
+<entry id="rrtmgp_enable_limiter_warnings" type="logical"
+       category="radiation" group="radiation_nl" valid_values="">
+Enables low temperature warnings in RRTMGP.
```

Comment:
See comment below, but right now this only applies to temperature warnings, but has a generic name of "rrtmgp_enable_limiter_warnings". Since we have limiters for other variables as well, I suggest either renaming or applying to the other limiters as well so that the description and name of this namelist variable are consistent and representative of what the code is doing.